tests/test_undate.py

```
@@ -1,13 +1,7 @@
-from datetime import timedelta, date
+from datetime import date, timedelta
```

Ruff

`datetime.timedelta` imported but unused

Remove unused import: `datetime.timedelta`

(F401)